### DISCUSSION TITLE
C functions

### DISCUSSION THREAD

**Discussion by MadFisa** [nschloe/quadpy#416] [Q&A]
Hi,

Is there any advantage in creating LowLevelCallable c functions for quadpy like in scipy integration routines? If yes can someone provide an example on how to do it? Thanks in advance.

**Comment by nschloe** [comment]
> Is there any advantage in creating LowLevelCallable c functions for quadpy like in scipy integration routines?

No. SciPy is already fully vectorized and the heavy lifting happens in the function evaluations. There is no benefit in moving any code to C.
